pkg/kubelet/cm/cpumanager/state/state_file.go

```diff
@@ -107,7 +106,8 @@ func (sf *stateFile) tryRestoreState() error {
 	}
 
 	for containerID, cpuString := range readState.Entries {
```

Comment:
I think we want to do the same thing here and move the `tmpContainerCPUSet` declaration outside of the function (i.e. similar to the change you made in `state_checkpoint.go`).

Reply:
I agree, I will make the shape of both functions more uniform, thanks!